tests/example-premature-termination/tst/example-premature-termination/example-premature-termination.cut

```diff
@@ -0,0 +1,5 @@
+TESTSUITE 'Testsuite example-premature-termination'
+
+TestCase "sucess 1"
+        PERFORM DO-SOMETHING
+	EXPECT WS-RESULT = "TRUE"
```

Comment:
```suggestion
	EXPECT WS-RESULT = "TRUE"
	
```

EOL

Reply:
Also, commit messages have mixed tense.  Commits do something in the moment, rather than in the past.

Not necessarily a "need to fix" but just be aware.

Might review Chris' Beam's article[1].

[1]: https://cbea.ms/git-commit/